common/src/stack/command/stack/commands/load/__init__.py

```diff
@@ -166,20 +166,23 @@ def load_firewall(self, firewalls, target=None):
 		scope = self.get_scope()
 		assert not (scope != 'global' and target is None)
 
-		for f in firewalls:
-			params = {'scope'         : scope,
-				  'service'       : f.get('service'),
-				  'network'       : f.get('network'),
-				  'output_network': f.get('output-network'),
-				  'chain'         : f.get('chain'),
-				  'action'        : f.get('action'),
-				  'protocol'      : f.get('protocol'),
-				  'flags'         : f.get('flags'),
-				  'comment'       : f.get('comment'),
-				  'table'         : f.get('table'),
-				  'name'          : f.get('name')}
-
-			self.stack('add.firewall', target, **params)
+		cmd = f"add.{f'{scope}.' if scope != 'global' else ''}firewall"
```

Comment:
f-string inside of an f-string feels wrong. You could do this to get the logic out of the f-string:
```
cmd = f'add.{scope}.firewall' if scope != 'global' else 'add.firewall'
```

Reply:
done.